src/ConnectedMode.UnitTests/ServerSentEvents/Issue/IssueServerEventsListenerTests.cs

```diff
@@ -63,7 +63,7 @@ public async Task OnEvent_SingleEvent_MultipleTuples_StoreIsUpdatedWithIssuesFro
             issueServerEventSource.Verify(x => x.GetNextEventOrNullAsync(), Times.Exactly(2));
             issueServerEventSource.VerifyNoOtherCalls();
 
-            storeUpdater.Verify(x => x.UpdateSuppressedIssues(
+            storeUpdater.Verify(x => x.UpdateSuppressedIssuesAsync(
```

Comment:
Add `Async` to method name (one of the interface methods had `Async` in the name, the other didn't)